dawn/main/MenuTemplate/DawnMenu.js

```diff
@@ -14,6 +15,14 @@ const DawnMenu = {
         app.quit();
       },
     },
+    {
+      label: 'Field Control Mode',
+      click() {
+        RendererBridge.reduxDispatch({
+          type: 'TOGGLE_FIELD_CONTROL',
```

Comment:
Is this toggle going to be a public facing toggle? What is the use case of students toggling between Field Control Mode and not. (Correct me if I'm completely misreading the code.